bundler/spec/dependabot/bundler/update_checker/force_updater_spec.rb

```diff
@@ -16,7 +16,7 @@
     described_class.new(
       dependency: dependency,
       dependency_files: dependency_files,
-      target_version: target_version,
+      target_version: Gem::Version.new(target_version),
```

Comment:
Were these tests previously using the wrong type in the setup and stubs?

Reply:
I think in the realworld a `Gem::Version` is always passed, so when I saw tests failing, I normalized the tests to also pass that.

I believe the `force_update` native helper (which this class mainly delegates to) works with a string or a `Gem::Version` indifferently, but my changes have broken that.

So, I could make this work for both `String` and `Gem::Version`, but since in the realworld a `Gem::Version` is passed, I figured it was passed to normalize the test to also pass that and let the class take a consistent type.

Reply:
Any opinions @jurre? I think the current version is fine, it will also make `ForceUpdater` easier to type I guess. Let me know if you prefer that I change it, though!

Reply:
No, seems like a good change